core/uvisor-config.h

```diff
@@ -17,7 +17,7 @@
 #ifndef  __UVISOR_CONFIG_H__
 
 #define UVISOR_MAGIC        0x2FE539A6
-#define UVISOR_FLASH_LENGTH 0x8000
+#define UVISOR_FLASH_LENGTH 0x8800
```

Comment:
0x9000 ? Should be uncritical as its kindof UVISOR_FLASH_MAX and wont waste space when not used.